package.json

```diff
@@ -8,19 +8,36 @@
     "eslint"
   ],
   "scripts": {
-    "lint": "eslint . --ext js"
+    "lint": "eslint . --ext js",
+    "missing": "npm run missing:js && npm run missing:ts",
+    "missing:js": "eslint-find-rules --verbose -u index.js",
+    "missing:ts": "eslint-find-rules --verbose -u ts.js | grep -v https://eslint.org/docs/rules/"
```

Comment:
This is why ts.js is separate from typescript.js.  It makes running `eslint-find-rules` output missing rules from `@typescript-eslint/eslint-plugin`.